config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+
+  get "login" => "user_sessions#new", as: :login
+  post "logout" => "user_sessions#destroy", as: :logout
+
+  get '/auth/twitter/callback', to: "user_sessions#create"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.